packages/data-schema/src/SchemaProcessor.ts

```diff
@@ -1316,6 +1386,64 @@ const mergeCustomTypeAuthRules = (
   }
 };
 
+function generateInputTypes(
```

Comment:
If we control the "InputTypes" naming that I think is introduced here, could we change it to "ArgumentTypes". Is there an important difference between arguments and inputs that I'm not seeing? If not, I think it would be better to stick to one name and I think arguments is the customer facing name.

Reply:
Alright. I made this comment when I understood less about the schema spec. Marking resolved.